lib/slimmer/skin.rb

```diff
@@ -10,7 +11,18 @@ def initialize options = {}
       @asset_host = options[:asset_host]
 
       @logger = options[:logger] || NullLogger.instance
-      @strict = options[:strict] || (%w{development test}.include?(ENV['RACK_ENV']))
+
+      @strict = options[:strict]
+      if %w{development test}.include?(ENV['RACK_ENV'])
+        # A Null instance logger passed in a lot,
+        # not too sure of the implications of unwinding
+        # this all as it depends on all apps that use slimmer.
+        # This will make sure everything is logged correctly
+        # and put slimer renderings more in line with rendering
+        # normal templates
```

Comment:
so the logger is dependent on what you pass through from the `Rails.application.config.slimmer` in the consuming app of slimmer